index.js

```diff
@@ -0,0 +1,185 @@
+     return;
+  } else {
+    addCustomerInputsToTopBar();
+    toggleDisplaySteps(step2, step1);
+  }
+}
+
+function showStepThree() {
+  collectPhoneData();
+  collectAccessoryData();
+  collectBundleData();
+
+  if(phoneChoice[0] === "" || accessoryChoice[0] === "" || bundleSize === "") {
+    alert('Please select one of each');
+    return;
+  } else {
+    combineDataForSummary();
+    moneyTotalsForSummary();
+  }
+  toggleDisplaySteps(step3, step2);
+}
+
+
+
+///
+//// NAVIGATION
+
+function toggleDisplaySteps(show, hide) {
+  show.classList.add("active");
+  hide.classList.remove("active");
+}
+
+
+
+///
+//// STEP 1
+
+function getCustomerInitialInputs() {
+  customerBankAccount = document.getElementById("bank-account").value;
+  customerSpendLimit = document.getElementById("spend-limit").value;
+}
+
+function addCustomerInputsToTopBar() {
+  document.getElementById('top-bar').innerHTML = `
+    <p id="input-balance">Estimated bank account: $${customerBankAccount}</p>
+    <p id="input-spend">Desired spend limit: $${customerSpendLimit}</p>`;
+}
+
+
+
+///
+//// STEP 2
+
+function collectPhoneData() {
+  phone = document.getElementById("phone-select");
+  phoneValue = phone.options[phone.selectedIndex].value;
+  phoneName = phone.options[phone.selectedIndex].text;
+
+  phoneChoice = [phoneValue, phoneName];
+  return phoneChoice;
+}
+
+function collectAccessoryData() {
+  accessory = document.getElementById("accessory-select");
+  accessoryValue = accessory.options[accessory.selectedIndex].value;
+  accessoryName = accessory.options[accessory.selectedIndex].text;
+
+  accessoryChoice = [accessoryValue, accessoryName];
+  return accessoryChoice;
+}
+
+function collectBundleData() {
+  bundleSize = document.getElementById("bundle-select");
+  bundleSize = parseInt(bundleSize.options[bundleSize.selectedIndex].value);
+
+  return bundleSize;
+}
+
+
+
+///
+//// STEP 3
+
+function customerInputsToNumbers() {
+  customerBankAccount = parseInt(document.getElementById("bank-account").value);
+  customerSpendLimit = parseInt(document.getElementById("spend-limit").value);
+}
+
+function overUnderForSummary() {
+  summarySpendTag = document.getElementById('summary-spent');
+
+  if(totalSpent < customerSpendLimit) {
+    summarySpendTag.innerHTML = `Great job! You are UNDER your spend limit by $${(customerSpendLimit - totalSpent).toFixed(2)}!`;
+  } else if(totalSpent === customerSpendLimit) {
+    summarySpendTag.innerHTML = `Nice planning, you have spent all of the money you set aside.`;
+  } else {
+    summarySpendTag.innerHTML = `Oh noes, you are OVER the spend limit by $${(totalSpent - customerSpendLimit).toFixed(2)}!`;
+  }
+}
+
+function dataPricesToNumbers() {
+  phonePriceFill = parseInt(phoneChoice[0]);
+  accessoryPriceFill = parseInt(accessoryChoice[0]);
+}
+
+function combineDataForSummary() {
+  const summaryPhoneTag = document.getElementById('summary-phone');
+  let phonePriceFill = phoneChoice[0];
```

Comment:
`let` declarations like this aren't necessary anymore - `phonePriceFill` is defined every time the function is run, so its safe to make it a constant. This should apply to every `let` inside of any of your functions.

Reply:
Good to know, I updated them to `const` and `UPDATED_THE_NAMES` in other places too

Reply:
Jk some of those mutated, put 'em back!